app/views/dams_objects/_admin_download.html.erb

```diff
@@ -4,6 +4,9 @@
 	etc_menu_item_class = 'pull-right'
 	embed_glyph =  '<i class="glyphicon glyphicon-share-alt"></i> Embed'.html_safe
 	adl_glyph =  '<i class="glyphicon glyphicon-download-alt"></i> Download File'.html_safe
+  file_format = (defined?(format)) ? "#{format.capitalize}" : ''
+  embed_width = (format == 'audio') ? '629' : '560'
```

Comment:
I'm curious about these embed widths and heights, was this part of the spec for this feature?

Reply:
@mcritchlow -  That's the width and height that Gabriela wanted.  I can add the test for width and height for image object.

Reply:
Ok sounds good, it seems quite specific.. which is why I was curious. Thanks